openedx/core/lib/api/permissions.py

```diff
@@ -80,3 +86,64 @@ def has_object_permission(self, request, view, obj):
         return (request.user.is_staff or
                 CourseStaffRole(obj.course_id).has_user(request.user) or
                 request.method in permissions.SAFE_METHODS)
+
+
+def get_profile_visibility(user_profile, user, configuration=None):
```

Comment:
@cahrens I just finished attempting this, and it also failed, for similar reasons. When permissions attempts to load the function from user_api.accounts_api, it fails because the api module depends on a module which depends on the permissions module. I haven't figured out which one yet, and it would take more time than we have.